### DISCUSSION TITLE
values are 2 days old

### DISCUSSION THREAD

**Discussion by bloodygizmo** [nielsmaerten/gluco-check#184] [Q&A] [answered]
i love this thing but my only problem is its giving me info "as of 2 days ago"

**Comment by nielsmaerten** [comment, marked as answer]
Are you uploading your pump's data to Nightscout as well? I went to check your site, but it looks like the most recent pump data available is about 2 days old.

When GlucoCheck reads out how old your values are, it takes the age of the oldest one in the set. So, if the most recent pump data is 2 days old... you'll hear "as of 2 days ago".

- If you don't want to upload pump data, just uncheck "Pump battery" and "Pump reservoir" in GlucoCheck's settings. That should fix the issue.
- If you DO want to have pump data in Nightscout, then it's possible your Mongo database is almost full. Go into your Nightscout's admin panel and remove some documents from the Devicestatus, Treatments and Entries collections.
